go.mod

```diff
@@ -7,4 +7,6 @@ require (
 	github.com/TexaProject/texajson v0.0.4
```

Comment:
After the texajson PR, I have rolled out a new release. So please make sure that your `go.mod` points to `v0.0.5`. :)

Reply:
sure :)